prow/plugins/approve/approvers/owners.go

```diff
@@ -565,6 +565,8 @@ Approval requirements bypassed by manually added approval.
 This pull-request has been approved by: {{range $index, $approval := .ap.ListApprovals}}{{if $index}}, {{end}}{{$approval}}{{end}}
 
 {{- if (and (not .ap.AreFilesApproved) (not (call .ap.ManuallyApproved))) }}
+To fully approve this pull request, please assign additional approvers. Any member of the organization can review and `+"`/lgtm`"+` a pull request, 
+and can be assigned to this PR, but in order for the pull request to be merged, it additionally has to be approved by a code owner listed in the OWNERS file for this repository(details below).
```

Comment:
This is text shouldn't be specific to our merge requirements. It should only describe the approval process which is just one of our merge requirements. The first sentence is fine, but `lgtm` may not always be required for merge.